android/src/main/java/com/rnfs/RNFSManager.java

```diff
@@ -75,15 +123,13 @@ public void writeFile(String filepath, String base64Content, ReadableMap options
   @ReactMethod
   public void readFile(String filepath, Callback callback) {
     try {
-      File file = new File(filepath);
-
-      if (!file.exists()) throw new Exception("File does not exist");
```

Comment:
I would also recommend not adding file:// without checking if the filepath already starts with file:// to avoid double file://file://